tests/libvmi/vmi.go

```diff
@@ -174,3 +174,54 @@ func baseVmi(name string) *v1.VirtualMachineInstance {
 	}
 	return vmi
 }
+
+// WithLabel sets labels with specified values
+func WithLabels(labels map[string]string) Option {
```

Comment:
This is a plural version of the existing singular option. Same for the annotations later.
Why would we want this?